public/scripts/asq.js

```diff
@@ -1004,7 +1004,7 @@ var Asq = {
 
 
   formatData: function(data) {
-    if (/[0-9a-z]{40}/i.test(data) || /[0-9]{4}\-[0-9]{2}\-[0-9]{2}/.test(data) || /[a-z]*/i.test(data)) {
+    if (/[0-9a-z]{40}/i.test(data) || /[0-9]{4}\-[0-9]{2}\-[0-9]{2}/.test(data) || (/[a-z]*/i.test(data) && !/^0\.[0-9]*E[0-9]*$/.test(data))) {
```

Comment:
Can you refactor this :hankey: a little bit to make it more readable while you're at it?